repo.yaml

```diff
@@ -15,6 +15,14 @@ repositories:
   metalink: https://mirrors.fedoraproject.org/metalink?repo=updates-released-f32&arch=ppc64le
   name: 32-ppc64le-update-repo
   gpgkey: https://src.fedoraproject.org/rpms/fedora-repos/raw/master/f/RPM-GPG-KEY-fedora-32-primary
+- arch: aarch64
+  metalink: https://mirrors.fedoraproject.org/metalink?repo=fedora-32&arch=aarch64
+  name: 32-aarch64-primary-repo
+  gpgkey: https://src.fedoraproject.org/rpms/fedora-repos/raw/master/f/RPM-GPG-KEY-fedora-32-primary
```

Comment:
`master` needs to be replaced with `main` now. Fedora switched to the `main` branch naming just a few days ago.